javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
ok, but maybe we leave this migration script as an option? We can describe it in release notes.
For large databases this kind of update could be really slow. For sure we need to add support for nulls (nulls can be treated as 0).

For users interested in new field, migration script (run manually) could update 
version sequences : [null, null, 1, 2] -> [1,2,3,4]

What do you think about this solution?